src/Tools/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.Codefix.Testing/CodeFixVerifier.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Analyzer.Testing;
+
+namespace Microsoft.CodeAnalysis.Codefix.Testing
+{
+    public class CodeFixVerifier<TAnalyzer, TCodeFix, TTest, TVerifier>
+           where TAnalyzer : DiagnosticAnalyzer, new()
+           where TCodeFix : CodeFixProvider, new()
+           where TTest : BaseCodeFixTest<TVerifier>, new()
+           where TVerifier : IVerifier, new()
+    {
+        public static DiagnosticResult[] EmptyDiagnosticResults
+            => DiagnosticVerifier<TAnalyzer, TTest, TVerifier>.EmptyDiagnosticResults;
+
+        public static DiagnosticResult Diagnostic(string diagnosticId = null)
+            => DiagnosticVerifier<TAnalyzer, TTest, TVerifier>.Diagnostic(diagnosticId);
+
+        public static DiagnosticResult Diagnostic(DiagnosticDescriptor descriptor)
+            => DiagnosticVerifier<TAnalyzer, TTest, TVerifier>.Diagnostic(descriptor);
+
+        public static DiagnosticResult CompilerError(string errorIdentifier)
+            => DiagnosticVerifier<TAnalyzer, TTest, TVerifier>.CompilerError(errorIdentifier);
+
+        public static Task VerifyDiagnosticAsync(string source, DiagnosticResult expected, CancellationToken cancellationToken = default)
```

Comment:
❓ Missing overload that takes `DiagnosticResult[]`?

Reply:
👍 I'll fix it